bids-validator/src/schema/expressionLanguage.ts

```diff
@@ -27,6 +27,15 @@ function exists(list: string[], rule: string = 'dataset'): number {
   } else {
     // dataset, subject and stimuli
     return list.filter((x) => {
+      /* local paths must start with slash. This leading slash will 
+       * trip up filetree.contains though so chomp if its present and fail if
+       * it isn't present.
+       */
+      if (x[0] == '/') {
+        x = x.substr(1)
+      } else {
+        return false
+      }
```

Comment:
This will break all the other datasets. Relative paths without initial slashes are extremely common. If we want to make this change, I would suggest getting input from a larger group first.

```suggestion
      // Leading slashes will trip up filetree.contains though so chomp if its present
      if (x[0] == '/') {
        x = x.substr(1)
      }
```